examples/organizational_cloudvision/versions.tf

```diff
@@ -4,5 +4,9 @@ terraform {
     aws = {
       version = ">= 3.50.0"
     }
+    sysdig = {
+      source = "sysdiglabs/sysdig"
+      version = "0.5.17" // TODO release updated sysdig provider
```

Comment:
Once https://github.com/sysdiglabs/terraform-provider-sysdig/pull/111 is merged, we will need to release this new version of the sysdig provider.